src/components/Activities/Activity1/Game2/components/SlideBox.tsx

```diff
@@ -0,0 +1,48 @@
+import React, { useState, useEffect } from 'react';
+
+import { PlaynetColors } from '../../../../shared/PlaynetConstants';
+
+interface SlideBoxProps {
+  handleClick ?: () => void;
+  imgSrc ?: string;
+}
+
+function SlideBox(props: SlideBoxProps): JSX.Element {
+  const BLACK : string = PlaynetColors.BLACK;
+  const HOVER_GREEN : string = PlaynetColors.HOVER_GREEN;
+  const INCORRECT_RED : string = PlaynetColors.INCORRECT_RED;
+
+  const imgSrc = props.imgSrc;
+
+  const [border, setBorder] = useState(BLACK);
+
+  useEffect (() => {
+    setBorder(BLACK);
+  }, [imgSrc]);
+
+  const handleHover = (color : string) => {
+    const newBorder = (border === INCORRECT_RED) ? INCORRECT_RED : color;
+
+    setBorder(newBorder);
```

Comment:
```suggestion
    if (border === PlaynetColors.INCORRECT_RED) return;
    setBorder(color);
```
try to avoid state changes!